Util/Emacs/README

```diff
@@ -1,2 +0,0 @@
-Emacs support for dafny is provided by the boogie-friends package, available from MELPA.
```

Comment:
Why did you delete this file? It seems like a good pointer. Is there another pointer to Emacs somewhere visible?

Reply:
Yes: in the README :)